aspnet/3-binary-data/Content/bootstrap-theme.css

```diff
@@ -0,0 +1,587 @@
+/*!
```

Comment:
Hmmm ... I wonder how these got in here.  I copy/pasted 2-structured-data.  I should test to see if VS adds these if they're missing, eg. for caching (unless you already know)

I'll remove these and add the path to .gitignore